qchem/tests/test_excitations.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
Is `import os` needed?

Reply:
Good catch!. Perhaps I used it before refactoring the function.

Reply:
```suggestion
```